portality/templates/doaj/toc.html

```diff
@@ -273,6 +273,11 @@ <h3 class="label label--underlined">Best practice</h3>
                                                target="_blank" rel="noopener">
                                                 {{ policy }}
                                             </a>
+                                        {% elif bibjson.deposit_policy_url %}
+                                            <a href="{{ bibjson.deposit_policy_url|safe }}"
+                                               target="_blank" rel="noopener">
+                                                {{ policy }}
+                                            </a>
```

Comment:
This is a good fall-back.  Did we find out if there is a URL for Mir@bel, or is this the way to handle it?

Reply:
Yes, I asked Dom on Slack and for now it is good,
![image](https://github.com/DOAJ/doaj/assets/8298769/d45afa60-1202-4009-b7bc-634b5a690d45)
@dommitchell  will check whether there is any specific link. If there is it will be quick hotfix.